### ISSUE TITLE
Is this API still useable?

### ISSUE THREAD

**Issue by Jvinniec** [kirkthaker/investopedia-trading-api#26]
Is this code still viable, given the fact that Investopedia has implemented a recaptcha on its simulator login page? I'm getting the following error message:

> InvestopediaApi.ita.LoginError: Login Error: Invalid Username or Password

I've double checked both my username and password, but it does not appear to work. Looking at the loggin page from a year ago shows that there wasn't a recaptcha, so maybe this is not working anymore?

**Comment by glotchimo** [comment]
The beauty of open source code is that you can just fork it and build in the updates yourself. My guess is that the existing auth protocol of this client doesn't work anymore if they updated it on the site's end. They have also since expanded their robots.txt file so this concept may no longer be viable.

**Comment by kirkthaker** [comment]
Yeah, this was bound to happen eventually if Investopedia changed their login flow. And if their robots.txt doesn't allow any kind of scraping/automated form submission, there's not much you can do (you should respect the robots.txt).
